integration_tests/dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'amplitude_source_integration_tests'
-version: '0.2.0'
+version: '0.2.1'
```

Comment:
Following my note from `stg_amplitude__event` I think this should be a breaking change. Let's increase to `0.3.0`